module/mouse_gutbrain_de_independent_guide.qmd

```diff
@@ -63,17 +77,17 @@ After you type the gene ID into the Search bar and hit enter, you should see a n
 ottrpal::include_slide("https://docs.google.com/presentation/d/1vikcyml6uw-_DaC1yFpS-UWoWNEGxFukMB_eCi42BqY/edit?slide=id.g346083d4dc0_0_0#slide=id.g346083d4dc0_0_0")
 ```
 
-:::{dictionary}
-You might come across some unexpected terms when you search the MGI for your gene ID. In addition to genes, Ensembl gene IDs are also given to “pseudogenes”, “putative genes”, and “lncRNA”.
+::: {dictionary}
```

Comment:
dictionary is not a callout available to quarto - you'll need one of these: https://quarto.org/docs/authoring/callouts.html